analyzer/src/analysis/value/interprocedural/sequential/function_fixpoint.cpp

```diff
@@ -91,7 +102,32 @@ FunctionFixpoint::FunctionFixpoint(Context& ctx,
       _checkers(caller._checkers),
       _exit_invariant(make_bottom_abstract_value(ctx)),
       _return_stmt(nullptr),
-      _logger(caller._logger) {}
+      _logger(caller._logger),
+      _namer(new ar::Namer(callee->body()))
```

Comment:
same here

Reply:
@arthaud, I have pushed a commit so that we only make the Namer if ar tracing is enabled.